docs/faq.rst

```diff
@@ -2,3 +2,30 @@
 
 FAQs
 ====
+
+How do I contribute to junifer?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Please follow the `contribution guidelines <https://juaml.github.io/junifer/main/contribution.html>`_.
```

Comment:
Can we use a sphinx link? I have the impression that this will link to the contribution guidelines in main. We have the multiversion plugin, so ideally it should link to the specific version link.

Reply:
I understand what you mean but I don't get how you link it to specific version.

Reply:
Ok, but at least lets use sphinx references and not the full URL path.

Reply:
Okay.

Reply:
Done.